alchemy-web/docs/providers/cloudflare/browser-rendering.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Managing Cloudflare Browser Rendering with Alchemy
+description: Learn how to use Cloudflare Browser Rendering with Alchemy for taking screenshots and automating browser tasks at the edge.
+---
+
+# BrowserRendering
+
+[Cloudflare Browser Rendering](https://developers.cloudflare.com/browser-rendering/) allows you to run a full browser instance within your Worker to take screenshots, generate PDFs, and automate browser tasks.
+
+## Minimal Example
+
+Create a basic Worker with browser rendering capabilities:
+
+```ts
+import { Worker, BrowserRendering } from "alchemy/cloudflare";
+
+const worker = await Worker("browser-worker", {
+  name: "browser-worker",
+  entrypoint: "./src/browser.ts",
+  compatibilityFlags: ["nodejs_compat"],
+  bindings: {
+    BROWSER: new BrowserRendering()
+  }
+});
+```
+
+## With KV Caching
+
+Create a Worker that takes screenshots and caches them in KV storage:
+
+```ts
+import { Worker, BrowserRendering, KVNamespace } from "alchemy/cloudflare";
+
+const cache = await KVNamespace("screenshot-cache", {
+  title: "Screenshot Cache"
+});
+
+const worker = await Worker("screenshot-service", {
+  name: "screenshot-service",
+  entrypoint: "./src/screenshot.ts",
+  compatibilityFlags: ["nodejs_compat"],
+  url: true,
+  bindings: {
+    BROWSER: new BrowserRendering(),
+    CACHE: cache
+  },
+  bundle: {
+    platform: "node"
+  }
+});
+```
+
+The Worker handler can use Puppeteer to control the browser:
+
+```ts
+// src/screenshot.ts
+import puppeteer from "@cloudflare/puppeteer";
+
```

Comment:
the example using KV namespace for caching is too complicated. Keep the example focused solely on browser rendering and a minimal example of how to use it in the worker runtime code